src/visitor/ExprVisitorImpl.cpp

```diff
@@ -125,5 +125,14 @@ void ExprVisitorImpl::visitBinaryExpr(BinaryExpression *expr) {
     }
 }
 
+void ExprVisitorImpl::visit(PathBuildExpression *expr) {
+    DCHECK(ok());
```

Comment:
it must true?

Reply:
Yes, the father expr should not call `accept` once it is not ok.